contracts/TokenSale.sol

```diff
@@ -443,7 +447,7 @@ contract TokenSale is OpsManaged, Pausable, TokenSaleConfig { // Pausable is als
 
 
     // Allows the admin to finalize the sale and complete allocations.
-    // The owner will also need to finalize the token contract so that
+    // The admin will also need to finalize the token contract so that
```

Comment:
Suggest being clear that  the `admin` that needs to `finalize` the token contract is `SimpleToken.admin` and not necessarily `TokenSale.admin`